syncd/scripts/syncd_init_common.sh

```diff
@@ -161,7 +161,11 @@ config_syncd_bcm()
       fi
 
     fi
-
+    
+    if [ -f "$HWSKU_DIR/context_config.json" ]; then 
+        CMD_ARGS+=" -x $HWSKU_DIR/context_config.json"
```

Comment:
when context config is passed, and there are multiple contexts defined, a "-g" parameter is needed to specify which context should be used, by default is "0" but if you have many syncds then it maybe different number

Reply:
Added "-g" option with id 0 for now.